modules/command/src/main/java/org/apache/fluo/command/BaseCommand.java

```diff
@@ -15,29 +15,14 @@
 
 package org.apache.fluo.command;
 
-import com.beust.jcommander.JCommander;
 import com.beust.jcommander.Parameter;
-import com.beust.jcommander.ParameterException;
-
-public class BaseOpts {
 
+public abstract class BaseCommand implements FluoCommand {
```

Comment:
I went back and forth on whether or not I should keep the options in the same class as the functionality for each command, or separate them into different classes. I ultimately decided that it would be more concise to have them be in the same class. Though I'm interested in other peoples opinions.

Reply:
I like the options in the same class.